seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/AbstractFlinkContainer.java

```diff
@@ -32,22 +31,19 @@
  * The before method will create a Flink cluster, and after method will close the Flink cluster.
  * You can use {@link AbstractFlinkContainer#executeJob} to submit a seatunnel config and run a seatunnel job.
  */
-@Slf4j
```

Comment:
I found out that all the integration tests had failed,and the reason is log be inherited.So i fix it provisionally.

@hailin0 @TyrantLucifer @ashulin

Reply:
> I found out that all the integration tests had failed,and the reason is log be inherited.So i fix it provisionally.
> 
> @hailin0 @TyrantLucifer @ashulin

@hailin0  PTAL

Reply:
> > I found out that all the integration tests had failed,and the reason is log be inherited.So i fix it provisionally.
> > @hailin0 @TyrantLucifer @ashulin
> 
> @hailin0 PTAL

#2946 has resolved this problem, these files should be reverted.

Reply:
> > > I found out that all the integration tests had failed,and the reason is log be inherited.So i fix it provisionally.
> > > @hailin0 @TyrantLucifer @ashulin
> > 
> > 
> > @hailin0 PTAL
> 
> #2946 has resolved this problem, these files should be reverted.

yes. I've revert.